dev/src/index.ts

```diff
@@ -755,8 +755,10 @@ export class Firestore {
    * @param bundleId. The id of the bundle. When loaded on clients, client SDKs use this id
    * and the timestamp associated with the built bundle to tell if it has been loaded already.
    * If not specified, a random identifier will be used.
+   *
+   * @private
    */
-  bundle(name?: string): BundleBuilder {
+  bundle_(name?: string): BundleBuilder {
```

Comment:
s/_bundle/bundle_

This is the much more common way to hide private APIs.

`snapshot_` is named `snapshot_` because it is actually part of the public interface since it is consumed by Firebase Functions.

Reply:
Done.